### DISCUSSION TITLE
Creating of Views (Mixed Read-only and Writeable Fields)

### DISCUSSION THREAD

**Discussion by pdesuze** [directus/directus#18813] [Feature Requests]
### Summary

Ability to create a view for a particular role with fields that they can modify

### Basic Example

View Project data but give approval (check box)

### Motivation

Internal Project Management Tool

### Detailed Design

Projects can be created by a Project Manager but the head of the department needs to approve.

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

None

### Alternatives

None

### Adoption Strategy

None

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
